tasks/help.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const ClientKitTask = require('clientkit-task');
+const Logr = require('logr');
+class HelpTask extends ClientKitTask {
+
+  constructor(name, config, runner) {
+    config.log = new Logr({
```

Comment:
I would imagine help is only run as standalone, so you don't really need it to stand out